### DISCUSSION TITLE
How can I enable the camera feed in OS X?

### DISCUSSION THREAD

**Discussion by ttorre1904** [hacksider/Deep-Live-Cam#417] [Q&A]
When I click "live", the Preview window appears. However, I see no camera feed on OBS or any other web meetings app (MS Teams, Zoom). How can I enable the camera feed, so that I can use the webcam mode in other applications?

I'm using an Apple Macbook M1 Pro on Mac OS 14.5.

**Comment by KaiD1993** [comment]
my question to you is how do you wanna use it if it doesn't work with M1 GPU ?

**Reply by ttorre1904** [reply]
I thought it was supported, using the [setup instructions](https://github.com/hacksider/Deep-Live-Cam?tab=readme-ov-file#coreml-execution-provider-apple-silicon) for CoreML Execution Provider (Apple Silicon)?

**Reply by ttorre1904** [reply]
To be clear the application is running and Preview shows, it just doesn't output any form of camera feed that I can tap into.

**Reply by hacksider** [reply]
You can capture the window from OBS and start from there.

**Reply by ttorre1904** [reply]
I did use OBS as an example but I'm actually not using that software. Are there any plans to supports OS X's virtual camera feed for easier integration?
